package.json

```diff
@@ -18,7 +18,7 @@
     "build": "yarn workspaces run build",
     "presetup": "yarn clean",
     "setup": "tsc --build",
-    "test": "export stage=DEV; yarn server test && yarn shared test",
+    "test": "export stage=DEV; yarn server test && yarn shared test && yarn modules test",
```

Comment:
Now that we're running tests in all of the workspaces, we can do 

```
"test": "export stage=DEV; yarn workspaces run test",
```

Reply:
nice!